pkgs/by-name/be/beeper-bridge-manager/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "bbctl";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "beeper";
+    repo = "bridge-manager";
+    rev = "v${version}";
```

Afaiks,

```suggestion
    rev = "refs/tags/v${version}";
```

is preferred now

---

Fixed on amended commit, thanks for the feedback!

---

Thanks for your quick response! Feel free to resolve this and the other threads (for some reason GitHub does not render `Resolve Thread` button for me).